doc/man3/SSL_get_event_timeout.pod

```diff
@@ -0,0 +1,87 @@
+=pod
+
+=head1 NAME
+
+SSL_get_event_timeout - determine when an SSL object next needs to be ticked
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_get_event_timeout(SSL *s, struct timeval *tv, int *is_infinite);
+
+=head1 DESCRIPTION
+
+SSL_get_event_timeout() determines when the SSL object next needs to perform
+internal processing due to the passage of time.
+
+All arguments are required; I<tv> and I<is_infinite> must be non-NULL.
+
+Upon the successful return of SSL_get_event_timeout(), one of the following
+cases applies:
+
+=over 4
+
+=item
+
+The SSL object has events which need to be handled immediately; I<*tv> is set to
+zero and I<*is_infinite> is set to 0.
```

Comment:
"set to zero" is slightly confusing since `tv` points to a struct type not an integer type.